pkg/controller/efs/filesystem/setup.go

```diff
@@ -67,11 +73,25 @@ func SetupFileSystem(mgr ctrl.Manager, o controller.Options) error {
 		Complete(r)
 }
 
+type custom struct {
+	kube     client.Client
+	client   svcsdkapi.EFSAPI
+	external *external
+
+	cache struct {
+		backupPolicy bool
```

Comment:
Boolean variables should always describe "yos-or-no" states. Better here would be `shouldUpdateBackupPolicy`.